src/map/mob.cpp

```diff
@@ -4055,6 +4055,8 @@ int mob_clone_spawn(map_session_data *sd, int16 m, int16 x, int16 y, const char
 		status->mode = mode;
 	else if (flag&1) //Friendly Character, remove looting.
 		status->mode = static_cast<enum e_mode>(status->mode&(~MD_LOOTER));
+	else if (flag&2) // Clone follows the creator, add MD_NORANDOMWALK
+		status->mode = static_cast<enum e_mode>(status->mode&MD_NORANDOMWALK);
```

Comment:
This one is wrong. Should be &~ like two lines above

Reply:
I think the intention was to add here, not remove.